src/main/java/com/_up/megastore/controllers/interfaces/ICategoryController.java

```diff
@@ -8,12 +8,18 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
+
+
 @RequestMapping("/api/v1/categories")
 public interface ICategoryController {
 
     @PostMapping @ResponseStatus(HttpStatus.CREATED)
     CategoryResponse saveCategory(@RequestBody @Valid CreateCategoryRequest createCategoryRequest);
 
+    @PostMapping(value = "/{categoryId}") @ResponseStatus(HttpStatus.OK)
+    CategoryResponse restoreCategory(@PathVariable UUID categoryId);
+
```

Comment:
Falta agregar la acción en el PATH que indica la razón de ser de este endpoint

```suggestion
    @PostMapping(value = "/{categoryId}/restore") @ResponseStatus(HttpStatus.OK)
    CategoryResponse restoreCategory(@PathVariable UUID categoryId);

```